docs/developer_guide/web_app_guide.md

```diff
@@ -1,66 +1,108 @@
-# Getting Started with Monero Javascript part 2: Creating a Web Application
+# Getting Started with Monero Javascript Part 2: Creating a Web Application
```

Comment:
The titles of other guide documents use lowercase after the first word.  Shall we change this or those to be consistent?

Reply:
Changing to lower. The Monero user guides and dev docs for other platforms use lower.